test/dom/ns-test.test.js

```diff
@@ -15,55 +14,44 @@ describe('XML Namespace Parse', () => {
 			'<html><body><c:var name="a" value="${1}"/></body></html>',
 			'text/xml'
 		)
-		var el = doc.getElementsByTagName('c:var')[0]
-		assert(el.namespaceURI, 'http://www.xidea.org/lite/core')
-		assert(
-			doc,
+		const el = doc.getElementsByTagName('c:var')[0]
+		expect(el.namespaceURI).toEqual('http://www.xidea.org/lite/core')
+		expect(doc.toString()).toEqual(
 			'<html xmlns="http://www.w3.org/1999/xhtml"><body><c:var name="a" value="${1}" xmlns:c="http://www.xidea.org/lite/core"></c:var></body></html>'
 		)
 	})
 
 	//ignore default prefix xml attribute
 	it('test', () => {
-		var w3 = 'http://www.w3.org/1999/xhtml'
-		var n1 = 'http://www.frankston.com/public'
-		var n2 = 'http://rmf.vc/n2'
-		var hx =
+		const w3 = 'http://www.w3.org/1999/xhtml'
+		const n1 = 'http://www.frankston.com/public'
+		const n2 = 'http://rmf.vc/n2'
+		const hx =
 			'<html test="a" xmlns="' +
 			w3 +
 			'" xmlns:rmf="' +
 			n1 +
 			'"><rmf:foo hello="asdfa"/></html>'
 
-		var doc = new DOMParser().parseFromString(hx, 'text/xml')
-		var els = [].slice.call(
+		const doc = new DOMParser().parseFromString(hx, 'text/xml')
+		const els = [].slice.call(
 			doc.documentElement.getElementsByTagNameNS(n1, 'foo')
 		)
-		for (var _i = 0, els_1 = els; _i < els_1.length; _i++) {
-			var el = els_1[_i]
+		for (let _i = 0, els_1 = els; _i < els_1.length; _i++) {
+			const el = els_1[_i]
 
-			var te = doc.createElementNS(n1, 'test')
+			let te = doc.createElementNS(n1, 'test')
 			te.setAttributeNS(n1, 'bar', 'valx')
-			var te = doc.createElementNS(n1, 'test')
```

Comment:
I think these 2 lines are redundant and should be removed, then we can use `const` instead of `let` here.

Reply:
I have removed the 2 redundant lines here.